### ISSUE TITLE
perl immediately hits "Illegal instruction" on launch

### ISSUE THREAD

**Issue by jckarter** [tbodt/ish#179]
If you install and run perl, it crashes pretty much immediately with an illegal instruction:

```
# apk add perl
...
# perl -v
Illegal instruction
# perl -h
Illegal instruction
# perl -e 'say "hi"'
Illegal instruction
```

I'll attach logs in a bit when I have my iPad at hand again.

**Comment by tbodt** [comment]
Fixed in 0a3498fa0a422b662d8a10723b62391c4a9da27e

**Comment by jckarter** [comment]
Thanks!
